src/datumaro/plugins/data_formats/roboflow/base.py

```diff
@@ -321,25 +319,38 @@ def _load_items(self, path):
         return items
 
 
-class RoboflowTfrecord(TfDetectionApiBase):
-    def __init__(
-        self,
-        path: str,
-        *,
-        subset: Optional[str] = None,
-        ctx: Optional[ImportContext] = None,
-    ):
-        super().__init__(
-            path=path, subset=subset, tfrecord_importer_type=TfrecordImporterType.roboflow, ctx=ctx
-        )
+try:
+    from datumaro.plugins.data_formats.tf_detection_api.base import TfDetectionApiBase
+    from datumaro.plugins.data_formats.tf_detection_api.format import TfrecordImporterType
+
+    class RoboflowTfrecord(TfDetectionApiBase):
```

Comment:
Please use `extra_deps("tensorflow")`:
https://github.com/openvinotoolkit/datumaro/blob/317fc1041941269908b03dbb478d1f0ee164a8e3/src/datumaro/components/lazy_plugin.py#L82-L101

It will prevent this plugin from being registered to `DEFAULT_ENVIRONEMENT`, so that an user will be disallowed to use it normally.

Reply:
Importing `TfDetectionApiBase` has make an error when tf is not installed, although `TfDetectionApiBase` tags extra_deps already.

Reply:
I have already tried to do 
    @extra_deps("tensorflow")
    class RoboflowTfrecordBase(TfDetectionApiBase):